doc/changelog.md

```diff
@@ -366,6 +366,10 @@
   the `stablehlo.custom_call` conversion pass.
   [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
+* Remove a duplicated variable definition for `grad_p` and add it to `func_p`, which was 
+  lacking from it 
+  [(#553)](https://github.com/PennyLaneAI/catalyst/pull/553)
+
```

Comment:
We usually add changelog entries for PRs that have user or development impacts. What do you think about removing this?

```suggestion
```

Reply:
Agree. Done.

Reply:
I don't think this is true. We don't add changelog entries that don't affect the _distributed_ package, for example updating the CI, build scripts, or documentation. But because we should also consider integrators of our package rather than just users, it can helpful to include non-user facing changes (it can be noted in the entry that it contains no user-facing changes though).

Reply:
Having said that, this change is so minimal that I'm fine merging it without an entry as well